### ISSUE TITLE
Allow multiple base patterns

### ISSUE THREAD

**Issue by skuzzle** [skuzzle/restrict-imports-enforcer-rule#7]
Allow multiple base patterns either like this:

```xml
<basePatterns>
  <basePattern>com.foo.**</basePattern>
  <basePattern>com.bar.**</basePattern>
</basePatterns>
```
or this 
```xml
<basePattern>com.foo.**, com.bar.**</basePattern>
```

Suggestion 1 is more consistent to the other configuration options while suggestion 2 is less verbose. Not sure what to favor.
Maybe even both ways might be allowed in every configurable entity which takes package patterns?

**Comment by skuzzle** [comment]
@aandeers Could you tell me what would be your favorite?

**Comment by aandeers** [comment]
Tough one. The first option is more consistent as you say but the second option is on the other hand more neat. Any way is good.

**Comment by skuzzle** [comment]
I've implemented the following: every configuration option which takes a package pattern can either take a list or a single pattern. You can now either write:

```xml
<basePackage>**</basePackage>
```

or

```xml
<basePackages>
    <basePackage>com.foo.**</basePackage>
    <basePackage>com.bar.**</basePackage>
</basePackages>
```

You can also specify both:
```xml
<basePackage>com.foo.**</basePackage>
<basePackages>
    <basePackage>com.bar.**</basePackage>
</basePackages>
```
In this case the single element is appended to the list.

Same for `bannedImports`, `allowedImports` and `excludedClasses`.

Will be released with v0.8.0.
